### PR TITLE
Add static site content

### PR DESCRIPTION
**Pull request by der**
The site needs to serve some static files for robot management and text to link to from QOS rejections.
Suggest these are served from the landing app so this PR adds the current production version of those files to the public assets.

Addresses: epimorphics/hmlr-ansible-deployment#109

### CODE REVIEW

File: public/qos-restriction.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+SPARQL queries, data API requests.
+
+Resource limitations for APIs intended to be used by the user interface are more restrictive
+than resource limitations on the data API and SPARQL queries.  If you are trying to retrieve
+data you are likely better off using the data API or SPARQL rather than scraping the UI pages
+or using UI specific APIs.
+
+See: 
+  http://landregistry.data.gov.uk/app/doc/ukhpi
+  http://landregistry.data.gov.uk/app/root/doc/ppd
```
Comment (jonrandahl) [inline public/qos-restriction.txt]: @der the links need the correct protocol in the uris:
- `https://landregistry.data.gov.uk/app/ukhpi/doc`
- `https://landregistry.data.gov.uk/app/root/doc/ppd`

File: public/qos-ui-api-restriction.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
+If you are seeing this it is because you have been programatically accessing a URL that
+it is intended for use only by HM Land Registry applications.
+
+If you wish to access HM Land Registry data in bulk, please download the bulk data file, e.g. at
+
+  https://www.gov.uk/government/statistical-data-sets/price-paid-data-downloads  
+
+If you wish to make adhoc queries of the HM Land Registry data, please use the documented data access APIs, e.g.
+
+  http://landregistry.data.gov.uk/app/root/doc/ppd
```
Comment (jonrandahl) [inline public/qos-ui-api-restriction.txt]: and here

File: public/qos-ui-api-restriction.txt [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  
+or the SPARQL endpoint at:
+
+  http://landregistry.data.gov.uk/landregistry/query
```
Comment (jonrandahl) [inline public/qos-ui-api-restriction.txt]: and here too

**Review by jonrandahl — CHANGES_REQUESTED** [review]
Some links need to be `https` not `http` as noted on the comments. 😅

File: public/qos-restriction.txt (hunk 1 above)
Comment (der) [reply public/qos-restriction.txt]: Fair enough, the sites all redirect to https but you are better to go there in the first place. Done.

File: public/qos-ui-api-restriction.txt (hunk 2 above)
Comment (der) [reply public/qos-ui-api-restriction.txt]: Done

File: public/qos-ui-api-restriction.txt (hunk 3 above)
Comment (der) [reply public/qos-ui-api-restriction.txt]: Done

**Review by jonrandahl — APPROVED** [review]
LGTM! 👍

**Pull request merged**